licenses/LICENSE-kittens.txt

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```

Comment:
Do we need this? CC0 is treated like public domain, as per http://www.apache.org/legal/resolved.html#can-works-placed-in-the-public-domain-be-included-in-apache-products . I found the image online that prompted this and it's also described as public domain.